scene/2d/tile_map.cpp

```diff
@@ -34,6 +34,36 @@
 
 #include "servers/navigation_server_2d.h"
 
+void TileMapCollider::set_body(RID p_body) {
+	body = p_body;
+}
+
+RID TileMapCollider::get_body() const {
+	return body;
+}
+
+void TileMapCollider::set_tile_map(TileMap *p_tile_map) {
+	tile_map = p_tile_map;
+}
+
+TileMap *TileMapCollider::get_tile_map() const {
+	return tile_map;
+}
+
+void TileMapCollider::set_coords(Vector2i p_coords) {
+	coords = p_coords;
+}
+
+Vector2i TileMapCollider::get_coords() const {
+	return coords;
+}
+
+void TileMapCollider::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("get_body"), &TileMapCollider::get_body);
+	ClassDB::bind_method(D_METHOD("get_tile_map"), &TileMapCollider::get_tile_map);
+	ClassDB::bind_method(D_METHOD("get_coords"), &TileMapCollider::get_coords);
```

Comment:
The preferred way to access read-only values is usually properties with getter only instead of just exposing getter methods (like in `KinematicCollision2D`).

Reply:
Is it ? I thought we wanted to get rid of read-only properties. I don't mind changing it but I'd rather be sure.

Reply:
You're right! I thought we were aiming at using properties rather than methods in general, but I didn't know gdscript didn't handle read-only properties. So I will rather remove the ones in physics :)